backend/utils/db_utils.py

```
@@ -52,6 +52,50 @@ def get_client_by_cf(cf):
     with get_db() as db:
         return db.query(Client).filter(Client.cf == cf).first()
 
+def _parse_date_field(value):
+    """Parse a date field value."""
+    if not value:
+        return None
+    try:
+        return datetime.strptime(value, "%Y-%m-%d")
+    except ValueError:
+        raise ValueError(f"Invalid date format: {value}") from None
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)